TandaHQ/workforcesuccess-frontend
# Changed position of Georgia and William. Updated Georgia's pic

Summary (non-technical)
--------
- Switched spots of Georgia and William
- Updated Georgia's pic

Problem & Solution (technical)
--------
- Switched spots of Georgia and William
- Updated Georgia's pic

Additional info
--------

QA Notes
--------
- Check if Georgia and William are in their designated spots.
- Check if Georgia has a real photo and not an illustration.
